server/lib/contributors.ts

```diff
@@ -99,7 +99,7 @@ const contributorsQuery = `
     c."isIncognito" as "isIncognito",
     BOOL_OR(COALESCE((c."data" ->> 'isGuest') :: boolean, FALSE)) AS "isGuest",
     COALESCE(MAX(m.description), MAX(tiers.name)) AS "description",
-    COALESCE(sum(transactions.amount) / count(DISTINCT m.id), 0) AS "totalAmountDonated"
+    COALESCE(sum(transactions."amountInHostCurrency" - transactions."paymentProcessorFeeInHostCurrency" - transactions."hostFeeInHostCurrency") / count(DISTINCT m.id), 0) AS "totalAmountDonated"
```

Comment:
For this metric, I wouldn't use the net amount. From the perspective of a financial contributor, if I contribute $10k to a collective, I expect the "Top contributors" section to me as $10k, not $9984.

```suggestion
    COALESCE(sum(transactions."amountInHostCurrency") / count(DISTINCT m.id), 0) AS "totalAmountDonated"
```

Reply:
You're correct Ben, I applied the logic to calculate the gross amount of the DEBIT transaction, even though we're explicitly selecting the CREDIT. This makes "amountInHostCurrency" the gross amount. Thanks for pointing out.

Reply:
@kewitz @Betree This is the opposite, we were explicitly adding back paymentProcessorFeeInHostCurrency and hostFeeInHostCurrency to make sure that this was the full amount. The previous code looked better.

If there is a doubt about it, can we cover this with a test?

Reply:
Disregard my comment. Except the part about test!